src/components/settings/OptionItem.tsx

```diff
@@ -7,27 +7,32 @@ import { InputNumberField } from "../form/InputNumberField";
 
 type OptionItemProps = {
   option: OptionsItem;
+  noOverlay?: boolean;
+  overlayId?: string;
 };
 
-export const OptionItem = ({ option }: OptionItemProps) => {
-  const { type, iD, description, label, shortname, value } = option;
+export const OptionItem = ({ option, noOverlay, overlayId }: OptionItemProps) => {
+  const { type, iD, description, label, shortname, value, overlayed } = option;
   const [optionValue, setOptionValue] = useState<string>("");
```

Comment:
optionValue isn't called anywhere. Therefore it could be removed.

Reply:
this is existing code, I don't wanna change that. I tried to remove it, but maybe to much... the settings were broken afterwards...